cmd/juju/application/addunit.go

```diff
@@ -24,51 +24,54 @@ var usageAddUnitSummary = `
 Adds one or more units to a deployed application.`[1:]
 
 var usageAddUnitDetails = `
-The add-unit command adds units to an existing application. It is used
-to scale out an application for improved performance or availability.
+The add-unit is used to scale out an application for improved performance or
+availability.
 
-Many charms will seamlessly support horizontal scaling while others
-may need an additional application support (e.g. a separate load
-balancer). See the documentation for specific charms to check how
-scale-out is supported.
+Many charms will seamlessly support horizontal scaling while others may need
+an additional application support (e.g. a separate load balancer). See the
+documentation for specific charms to check how scale-out is supported.
 
-By default, units are deployed to newly provisioned machines in
-accordance with any application or model constraints. This command
-also supports the placement directive ("--to") for targeting specific
-machines or containers, which will bypass application and model
-constraints.
+By default, units are deployed to newly provisioned machines in accordance
+with any application or model constraints. This command also supports the
+placement directive ("--to") for targeting specific machines or containers,
+which will bypass application and model constraints.
 
 Examples:
 
 Add five units of wordpress on five new machines:
+
     juju add-unit wordpress -n 5
 
 Add a unit of mysql to machine 23 (which already exists):
+
     juju add-unit mysql --to 23
 
 Add two units of mysql to machines 3 and 4:
```

Comment:
do machines 3 & 4 already exist?  is mentioned in example above.

Description seems ambiguous?  "Add two units of mysql, one each, to machines 3 and 4:"?

Reply:
Thanks, noted that they're "existing machines"